From 7831caa1eaef85e1e22ceabcccdc167e54f387c9 Mon Sep 17 00:00:00 2001
From: fake <fake@foo.cn>
Date: Sun, 8 Mar 2020 07:51:55 +0000
Subject: [PATCH pytorch] third_party/sleef: add support mips arch in v1.4.0

---
 src/arch/helperpurec_scalar.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/arch/helperpurec_scalar.h b/src/arch/helperpurec_scalar.h
index 9c11ace..5565577 100644
--- a/src/arch/helperpurec_scalar.h
+++ b/src/arch/helperpurec_scalar.h
@@ -43,7 +43,7 @@
 #define ENABLE_FMA_DP
 #define ENABLE_FMA_SP

-#if defined(__AVX2__) || defined(__aarch64__) || defined(__arm__) || defined(__powerpc64__)
+#if defined(__AVX2__) || defined(__aarch64__) || defined(__arm__) || defined(__powerpc64__) || defined(__mips__)
 #ifndef FP_FAST_FMA
 #define FP_FAST_FMA
 #endif
--
2.20.1

